health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/NewCampaignCreate/CampaignHome.js

```
@@ -0,0 +1,113 @@
+          }}
+        >
+          <SVG.SystemUpdateAlt width="40" height="40" fill={"#505a5f"} />
+          <div className={"descStyle"}>{t("HCM_IMPORT_EXISTING_CAMPAIGN")}</div>
+        </div>
+      </div>
+      {showPopUp && (
+        <PopUp
+          // className={"boundaries-pop-module"}
+          type={"default"}
+          heading={t("CAMPAIGN_REQUIREMENTS_HEADING")}
+          children={[]}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)